content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,173 @@
+
+<pre>
+<code>
+DD_BETA_COMMANDS_ENABLED=true DATADOG_SITE={{< region-param key="dd_site" >}} DATADOG_API_KEY=&lt;API_KEY&gt; DATADOG_APP_KEY=&lt;APP_KEY&gt; datadog-ci gate evaluate
+</code>
+</pre>
+
+The behavior of the command can be modified using the following flags:
+- `--fail-on-empty`: when this flag is specified, the command fails if no matching rules were found in Datadog
+based on the current pipeline context. By default, the command succeeds.
+- `--fail-if-unavailable`: when this flag is specified, the command fails if one or more rules could not be evaluated because of an internal issue.
+By default, the command succeeds.
+- `--no-wait`: by default, the command waits a certain amount of time for the events (tests, static analysis violations) to arrive to Datadog.
+This step is important as it makes sure that the events are queryable in Datadog before the rules are executed,
+avoiding incorrect evaluations. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is
+called several minutes after the related events are sent to Datadog, you could skip this waiting time by specifying the `--no-wait` flag.
+Note that, if used incorrectly, this flag might result in inaccurate rule evaluations.
+
+[Custom scopes](#custom-scope) can be added by using the following option one or more times: `--scope`:
+
+{{< code-block lang="shell" >}}
+datadog-ci gate evaluate --scope team:backend --scope team:frontend
+{{< /code-block >}}
+
+The command logs can be examined to understand what was the overall gate evaluation status, along with information
+about all the rules that were evaluated.
+
+{{< img src="ci/datadog_ci_gate_evaluate_logs.png" alt="Datadog-ci gate evaluate logs" style="width:90%;">}}
+
+## Rule scope
+
+When creating a rule, you can define a scope, which states when the rule should be evaluated. If the rule scope is
+set to `always evaluate`, the rule is evaluated on all repositories and branches.
+
+{{< img src="ci/rule_scope_always_evaluate.png" alt="Rule scope for rules always evaluated" style="width:90%;">}}
+
+When the `datadog-ci gate evaluate` command is invoked, the rules having a scope matching the command context are evaluated.
+
+For each scope (for example, `branch`), you can either select included or excluded values.
+When included values are selected, the rule is evaluated if one or more included values are part of the command context.
+When excluded values are selected, the rule is evaluated if the excluded values are not part of the command context.
+
+For example, to create a rule that is evaluated in all branches but `main` of the `example-repository` repository, you can define the following scope:
+
+{{< img src="ci/scope_not_main_example_repository.png" alt="Rule scope for example-repository and not main branch" style="width:90%;">}}
+
+If a rule does not contain a scope, it is evaluated for all values for that scope.
```

Comment:
to make this clearer I would say, "If a rule doesn't explicitly include a certain type of scope, it is evaluated for any value of that particular type."

Reply:
It's weird to see the word "type" here. Maybe scope name? I think in general the next sentence/example makes it clear